.github/workflows/windows.yml

```
@@ -0,0 +1,83 @@
+      - name: Build the project
+        run: |
+          cd build
+          cmake --build . --config Debug
+
+      - name: Copy header files and library
+        run: |
+          New-Item -ItemType Directory -Force -Path "${{ github.workspace }}\bin\Debug\include\GUI\components"
+          New-Item -ItemType Directory -Force -Path "${{ github.workspace }}\bin\Debug\include\GUI\form"
+          New-Item -ItemType Directory -Force -Path "${{ github.workspace }}\bin\Debug\include\XML"
+          New-Item -ItemType Directory -Force -Path "${{ github.workspace }}\bin\Debug\lib"
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)